src/ClientData/CaptureData.cpp

```diff
@@ -113,10 +114,7 @@ void CaptureData::AddFunctionStats(uint64_t instrumented_function_id,
 void CaptureData::OnCaptureComplete(
     const std::vector<const orbit_client_data::TimerChain*>& chains) {
   // Recalculate standard deviation as the running calculation may have introduced error.
-  for (auto& pair : functions_stats_) {
-    FunctionStats& stats = pair.second;
-    stats.set_variance_ns(0);
-  }
+  std::unordered_map<int, unsigned long> id_to_sum_of_deviations_squared;
```

Comment:
We prefer `absl::flat_hash_map` to `std::unordered_map`, it's a drop-in replacement with better performance.

Reply:
done. thanks!